### DISCUSSION TITLE
Unable to download dependency

### DISCUSSION THREAD

**Discussion by miluren** [Eiriksgata/mirai-rulateday-dice#132] [Q&A]
Unresolved dependency: 'com.github.Eiriksgata:trpg-java-dice:jar:1.3.1
I pulled down Mirai's dice plugin and used this dependency1'

**Comment by eiriksgata** [comment]
https://github.com/Eiriksgata/trpg-java-dice

https://www.jitpack.io/#Eiriksgata/trpg-java-dice
